aruna/api/storage/services/v1/collection_service.proto

```diff
@@ -104,8 +104,6 @@ message CreateNewCollectionRequest {
   string name = 1;
   // Description
   string description = 2;
-  // Project id
-  string project_id = 3;
```

Comment:
Is it really a good idea to remove the `project_id` parameter from the `CreateNewCollectionRequest`? I don't think so...

Reply:
Fixed by https://github.com/ArunaStorage/ArunaAPI/pull/44/commits/df320a150f1a460dc0dc3a7027234337dcb96e6b